SimCalorimetry/EcalElectronicsEmulation/src/EcalSimRawData.cc

```diff
@@ -555,31 +555,34 @@ void EcalSimRawData::getTp(const edm::Event &event, const std::string &collName,
       }
     }
     if (tpVerbose_) {
-      cout << setfill('0');
+      std::cout << std::setfill('0');
     }
     for (EcalTrigPrimDigiCollection::const_iterator it = tpDigis.begin(); it != tpDigis.end(); ++it) {
       const EcalTriggerPrimitiveDigi &tp = *it;
       int iTtEta0 = iTtEta2cIndex(tp.id().ieta());
       int iTtPhi0 = iTtPhi2cIndex(tp.id().iphi());
       if (iTtEta0 < 0 || iTtEta0 >= nTtEta) {
-        cout << "iTtEta0 (= " << iTtEta0 << ") is out of range ("
-             << "[0," << nEbTtEta - 1 << "])\n";
+        std::cout << "iTtEta0 (= " << iTtEta0 << ") is out of range ("
+                  << "[0," << nEbTtEta - 1 << "])\n";
       }
       if (iTtPhi0 < 0 || iTtPhi0 >= nTtPhi) {
-        cout << "iTtPhi0 (= " << iTtPhi0 << ") is out of range ("
-             << "[0," << nTtPhi - 1 << "])\n";
+        std::cout << "iTtPhi0 (= " << iTtPhi0 << ") is out of range ("
+                  << "[0," << nTtPhi - 1 << "])\n";
       }
 
       tcp[iTtEta0][iTtPhi0] = tp[tp.sampleOfInterest()].raw();
 
       if (tpVerbose_) {
-        if (tcp[iTtEta0][iTtPhi0] != 0)  // print non-zero values only
-          cout << collName << (collName.empty() ? "" : " ") << "TP(" << setw(2) << iTtEta0 << "," << iTtPhi0 << ") = "
-               << "0x" << setw(4) << tcp[iTtEta0][iTtPhi0] << "\tcmssw indices: " << tp.id().ieta() << " "
-               << tp.id().iphi() << "\n";
+        if (tcp[iTtEta0][iTtPhi0] != 0) {  // print non-zero values only
+          std::string collName = (type == 0) ? tcpDigiCollection_ : tpDigiCollection_;
```

Comment:
```suggestion
          std::string collName = (type == EcalSimRawData::tcp) ? tcpDigiCollection_ : tpDigiCollection_;
```
(Very minor point, not worth preventing this to get merged, but perhaps you'll like to take it into account in further cleanups)